js/modules/cohortbuilder/components/ConditionOccurrenceTemplate.html

```diff
@@ -49,7 +49,7 @@
 			<tr data-bind="if: ConditionType() != null, visible: ConditionType() != null">
 				<td><i data-bind="click: function() { $component.removeCriterion('ConditionType') }" class="fa fa-times" /></td>
 				<td>
-					Condition Type is:
+					Condition Type <button class="btn btn-primary btn-xs" data-bind="click: function() { ConditionTypeExclude(!ConditionTypeExclude())}, text: ConditionTypeExclude() ? 'is not any of': 'is any of'"></button> 
```

Comment:
The added button is pretty the same in all changed files: wouldn't it be reasonable to move it into a separate template which accepts boolean observable?

Reply:
We can do that.